resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/useCreateAchievementTicketForm.ts

```diff
@@ -0,0 +1,96 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { toastMessage } from '@/common/components/+vendor/BaseToaster';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { TicketType } from '@/common/utils/generatedAppConstants';
+
+const createAchievementTicketFormSchema = z.object({
+  /** @see TicketType.php */
+  issue: z.enum(['DidNotTrigger', 'TriggeredAtWrongTime', 'NetworkIssue']),
+  emulator: z.string().min(1),
+  emulatorVersion: z.string().optional(),
+  core: z.string().optional(),
+  mode: z.enum(['hardcore', 'softcore']),
+  hash: z.string().min(1),
+  description: z
+    .string()
+    .min(25, { message: 'Please be more detailed in your description.' })
+    .refine((desc) => !/(n'?t|not?).*(work|trigger)/i.test(desc), {
+      message: 'Please be more detailed in your description.',
+    })
```

Comment:
"doesn't work" may be valid. I think the old logic had a minimum length check too.

> This doesn't appear to work if you haven't been to see the king.

Reply:
True - similar to the existing page, a warning will now display, but the form will not be invalidated.